packages/@o3r/rules-engine/src/engine/fact/current-time/current-time-facts.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+import { FactsService } from '../../../fact';
+import { RulesEngineService } from '../../../services';
+import { CurrentTimeFacts } from './current-time.facts';
+
+@Injectable({
```

Comment:
An Angular Injectable should not be in the `engine` folder.
I do even think that this fact implementation should not be in the Engine module at all